### PR TITLE
Feature/update behat config

### PR DESCRIPTION
**Pull request by turegjorup**

### CODE REVIEW

File: tests/Behat/FixturesContext.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        $this->elasticService = $elasticService;
+
+        // @TODO: To be refactored when Fos\Elastica is removed.
+        $this->configManager = $configManager;
+        $this->indexManager = $indexManager;
+        $this->resetter = $resetter;
+    }
+
+    /**
+     * @Given the following search entries exits:
```
Comment (tuj) [inline tests/Behat/FixturesContext.php:48]: exits => exists

File: tests/Behat/FixturesContext.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+     *
+     * @param TableNode $table
+     *   Gherkin table argument containing columns identifier, type, url, autogenerated, image_format
+     */
+    public function theFollowingIdentifiersExits(TableNode $table): void
```
Comment (tuj) [inline tests/Behat/FixturesContext.php:53]: Exits => Exists

**Comment by cableman** [comment]
Should the "apcu" extension be added to github workflows. Not should I understand why they don't file or is it only required in production mode? It should be added to the `tag.yml` file to ensure the build process works?

**Comment by turegjorup** [comment]
> Should the "apcu" extension be added? [...] It should be added to the `tag.yml` file 

`apcu` added in `pr.yml` - this is the only place we configure php extensions as far as I can see.

**Pull request merged**